src/csv/builder.cr

```diff
@@ -111,7 +111,7 @@ class CSV::Builder
     end
 
     # Appends the given value to this row.
-    def <<(value : String)
+    def <<(value : String) : Nil
```

Comment:
It's my expectation that `<<` should return self. `Array#<<` does this since it is an alias of `Array#push`

Reply:
Agreed. That's similar to https://github.com/crystal-lang/crystal/pull/10622#issuecomment-817838293 I'd prefer to keep this out of thiss PR and add method chaining features separately. This keeps the process more focused.